src/WL/AppBundle/Lib/Breadcrumbs/BreadcrumbsBuilder.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jjuszkiewicz
+ * Date: 05.09.2014
+ * Time: 10:59
+ */
+
+namespace WL\AppBundle\Lib\Breadcrumbs;
+
+
+use Nokaut\ApiKit\Entity\Category;
+use Nokaut\ApiKit\Ext\Data\Collection\Filters\FiltersAbstract;
+use Nokaut\ApiKit\Ext\Data\Entity\Filter\FilterAbstract;
+use WL\AppBundle\Lib\Type\Breadcrumb;
+
+class BreadcrumbsBuilder
```

Comment:
I this a BreadcrumbsAllowedCategoriesBuilder and BreadcrumbsBuilder are best example for inheritance model. Both has public method p(repareBreadcrumbs, appendFilter), but in BreadcrumbsAllowedCategoriesBuilder method prepareBreadcrumbs has to do somethings more.